omod/src/main/java/org/openmrs/module/patientcolors/web/controller/PatientcolorsController.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.patientcolors.web.controller;
+
+import java.util.List;
+
+import javax.servlet.http.HttpSession;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.User;
+import org.openmrs.api.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+/**
+ * This class configured as controller using annotation and mapped with the URL of
+ * 'module/${rootArtifactid}/${rootArtifactid}Link.form'.
+ */
+@Controller("${rootrootArtifactid}.PatientcolorsController")
```

Comment:
change all the occurrences of `${rootrootArtifactid}` to `patientcolors`

Reply:
thanks man gonna work on it

Reply:
actually first try correcting `${rootrootArtifactid}` to` ${rootArtifactid}`. only here , and build the module again and install it , and see if its running

Reply:
first wait to implement this  

>  change all the occurrences of ${rootrootArtifactid} to patientcolors